<ui version="4.0" >
 <class>KwSongdbSongBooksEditWidget_base</class>
 <widget class="QWidget" name="KwSongdbSongBooksEditWidget_base" >
  <property name="windowTitle" >
   <string>Edit Song Books</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" native="1" name="widget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" name="layoutSongBooksToolBar" />
       </item>
       <item>
        <widget class="QListWidget" name="listSongBooks" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupSongBook" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Song Book</string>
      </property>
      <layout class="QFormLayout" name="formLayout" >
       <property name="fieldGrowthPolicy" >
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0" >
        <widget class="QLabel" name="labelAbreviation" >
         <property name="text" >
          <string>&amp;Abreviation</string>
         </property>
         <property name="buddy" >
          <cstring>editAbreviation</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="labelDescription" >
         <property name="text" >
          <string>&amp;Description</string>
         </property>
         <property name="buddy" >
          <cstring>editDescription</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="KLineEdit" name="editAbreviation" />
       </item>
       <item row="3" column="1" >
        <widget class="QPlainTextEdit" name="editDescription" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labelName" >
         <property name="text" >
          <string>&amp;Name</string>
         </property>
         <property name="buddy" >
          <cstring>editName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="KLineEdit" name="editName" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editAbreviation</tabstop>
  <tabstop>editName</tabstop>
  <tabstop>editDescription</tabstop>
  <tabstop>listSongBooks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
